srcpkgs/gama/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gama'
+pkgname=gama
+version=1.2.1
+revision=1
+build_style=go
+go_import_path="github.com/termkit/gama"
+makedepends="make git pkg-config"
+short_desc="Manage your GitHub Actions from Terminal with great UI"
+maintainer="zenobit <zenobit@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/termkit/gama"
+distfiles="https://github.com/termkit/gama/archive/refs/tags/v${version}.tar.gz"
+checksum=d2fad1280142b0cc8cb311a5e328590feb0c5a1642c47e3f8e0aaf1b713f6c7b
+
+do_build() {
+	make build
```

Comment:
Try to get `build_style=go` to do the build so that `do_install` from it also works.